addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -194,6 +204,9 @@ odoo.define('website.s_website_form', function (require) {
             // All 'hidden if' fields start with d-none
             this.$target[0].querySelectorAll('.s_website_form_field_hidden_if:not(.d-none)').forEach(el => el.classList.add('d-none'));
 
+            // Reset the initial default values.
+            this.initialValues.forEach(el => el.fieldEl.setAttribute('value', el.initialValue));
+
```

Comment:
I was already scared that you register all initials values (even the ones that are not prefilled) but if you do it this way, I am pretty sure this will leave `value=""` on every field at reedition... :grimacing: Or am I wrong?